src/modules/navigator/navigator_main.cpp

```diff
@@ -684,17 +684,16 @@ Navigator::get_cruising_speed()
 {
 	/* there are three options: The mission-requested cruise speed, or the current hover / plane speed */
 	if (_vstatus.is_rotary_wing) {
-		if (_mission_cruising_speed_mc  > FLT_EPSILON
+		if (_mission_cruising_speed_mc  > 0.0f
```

Comment:
If I recall correctly there aren't floating calculations done on this variable, it's just the compiler checks forcing the warnings for equality tests. For < or > tests we aren't ever floating around near zero values so just doing a normal check is fine. But other than that, no need for the change.